tests/integration/helpers.py

```diff
@@ -1,16 +1,165 @@
+#!/usr/bin/env python3
 # Copyright 2021 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-import logging
+from typing import List, Literal
 
-log = logging.getLogger(__name__)
+import aiohttp
+from pytest_operator.plugin import OpsTest
 
 
-async def get_unit_address(ops_test, app_name: str, unit_num: int) -> str:
-    status = await ops_test.model.get_status()  # noqa: F821
+class Prometheus:
```

Comment:
```suggestion
class PrometheusClient:
```

Reply:
:-1: for a number of reasons:

* It's a helper in an integration test and the name doesn't matter that much as long as we know what it means
* `PrometheusClient | prometheus_client` is the naming for the official "this application provides a metrics endpoint and this `PrometheusClient | prometheus_client` is full of methods for creating graphs/vectors/etc.
* This code is directly from `tests/integration/helpers.py` in `prometheus-k8s`, and we don't need to diverge there

Reply:
You are correct that this is taken directly from the prometheus repo including the class name. We will keep it as is to match that unless there are major objections.